nativelib/src/main/scala/scala/scalanative/meta/LinktimeInfo.scala

```diff
@@ -24,6 +24,9 @@ object LinktimeInfo {
   @resolvedAtLinktime("scala.scalanative.meta.linktimeinfo.isFreeBSD")
   def isFreeBSD: Boolean = resolved
 
+  @resolvedAtLinktime("scala.scalanative.meta.linktimeinfo.arch")
+  def arch: String = resolved
```

Comment:
Another possible name: `targetArch`?

Reply:
how about a dedicated object?
```scala
object target {
 def arch: String = resolved 
 // in the future
 def subArch: String = resolved
 def vendor: String = resolved
 def environment: String = resolved
}
```

Reply:
I fixed my comments below - we could see what LLVM names they to keep consistent.